Xamarin.Forms.Controls/GalleryPages/CarouselViewGalleryPage.xaml.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System;
+using System.Collections.ObjectModel;
+using System.Diagnostics;
+using System.Linq;
+
+namespace Xamarin.Forms.Controls
+{
+	public partial class CarouselViewGalleryPage : ContentPage
```

Comment:
It's customary to automate the UITests. Please add labels which expose all the bindable properties instead of dumping to the debug log (e.g. position, ItemSource.Length, etc). This will allow you to add assertions in the UITest that the values are as you expect them to be. E.g. that when an item is inserted to the left of the current item that the Position BP is bumped (leaving the current item unchanged) and when to the right is not. In general, UITests should be added for every button given various states of the CarouselView (number of elements/position).